src/locales/pull.sh

```diff
@@ -4,7 +4,10 @@ set -e
 # Usage:
 #   - Create a READ ONLY a token at https://app.lokalise.com/profile
 #   - Run the command from the root of the project with:
-#     LOKALISE_TOKEN=<token> yarn i18n:pull
+#     LOKALISE_READ_TOKEN=<token> yarn i18n:pull
+#   - To specify which languages do pull add the LANGUAGES with comma separated
+#     locale strings:
+#     LANGUAGES=en,es_PR,el yarn i18n:pull
```

Comment:
This is in order to filter the languages we support on the build only, otherwise we will add translation files to each one of the languages available on lokalise for android and iOs

Reply:
Makes sense to me. But to make sure i fully understand the trade off, would there be any consequence to including all of the language files in lokalise, other than unused code?

And if the languages files are not being used in the app, should we stop translating them in lokalise? Seems like unnecessary work for the translation team? And this would also allow us to avoid needing the extra flags in the script.

Reply:
I see your point, I just don't know what the intent is on having all the translations available. It might be an organizational decision of having all the translations available for when they are used, this just increases the bundle size a little bit I don't think is such a big deal, I just added it to make sure we can update the ones that are ready. Since I believe that if a translation file is present on iOS and the user phone is using that language it will load it, and I'm not sure what the implications are for having the wrong translation vs defaulting to English.

Reply:
I made it so the extra flags are not required but I felt we might need them

Reply:
This makes sense to me too. If the translations for languages not yet included in any release are adding to the bundle size, there is no need to have the languages folder. If it helps and makes code less complicated, I can exclude the languages not needed before creating the pull request.
We can continue to have translations for such languages in Lokalise as there's no cost associated with it and it helps to have translations done regularly when we have the translators (example, we already had Russian and Spanish ready and updated for a while before they were actually needed for a release). But we can flag them or exclude them in the PR.